tests/webapp/api/test_performance_data_api.py

```diff
@@ -709,3 +711,37 @@ def test_filter_out_retriggers():
 
     filtered_data = PerformanceSummary._filter_out_retriggers(copy.deepcopy(no_retriggers_data))
     assert filtered_data == no_retriggers_data
+
+
+def test_alert_tasks_get(client, test_perf_alert_summary, test_perf_data):
+    create_perf_alert(
+        summary=test_perf_alert_summary,
+        series_signature=test_perf_data.first().signature,
+        related_summary=test_perf_alert_summary,
+        status=PerformanceAlert.REASSIGNED,
+    )
+    resp = client.get(
+        reverse('performance-alertsummary-tasks') + '?id={}'.format(test_perf_alert_summary.id)
+    )
+    assert resp.status_code == 200
+    assert resp.json() == {
+        'id': test_perf_alert_summary.id,
+        'tasks': [
+            'Mochitest Browser Chrome',
+            'Inari Device Image Build',
+            'B2G Emulator Image Build',
+            'Nexus 4 Device Image Build',
+        ],
+    }
+
+    not_exist_summary_id = test_perf_alert_summary.id
```

Comment:
@junngo could you move these two failure case tests to a separate unit test?

Reply:
Okay :) I separated the failure case and submitted the pr[0] again.

[0]
https://github.com/mozilla/treeherder/pull/7717/files#diff-3fc99121ba535a30fbab26c4f861348f58cba0a54645e236a07619e1fe3df183R738-R750